### ISSUE TITLE
Closing DDLC on linux

### ISSUE THREAD

**Issue by carepollo** [Monika-After-Story/MonikaModDev#8505]
I use Ubuntu 20.04.3 and whenever I say goodbye to monika  this happens 
![screenshot](https://cdn.discordapp.com/attachments/703071798765420604/923054045055291392/unknown.png)

The program itself stops working and I also tried clicking wait many times but it just keeps prompting up again and again.
I am not sure how to reproduce this considering that the last time I opened MAS on linux this was not a problem.

**Comment by Batcastle** [comment]
Not having this issue on Drauger OS 7.5.1 or 7.6 Beta 1 (based on Ubuntu 20.04 and 22.04 (development) respectively). I suggest starting out with making sure your system is up-to-date. 

Maybe try on another distro from a live USB? Just make sure you copy the `.renpy` folder in your home directory to the home directory of the live USB so that your save data is picked up (copy, not cut. Don't want to lose your Monika!)

**Comment by multimokia** [comment]
Haven't seen this either coming from Manjaro, kernel vers `5.15.2-2`. However I will note that #8111 may end up addressing this issue for you.

**Comment by carepollo** [comment]
Well, I have to point out that I have copied a regular DDLC instalation from Linux and deployed it into my Linux environment, then I made a little change on DDLC. Py (nothing relevant just changing the persistant file location) and then I imported my previously backed up persistant folder.

I didn't mention because I doubted it was important but I see that the link above talks about it, for now I'm going to read it and see how it goes.

**Comment by ThePotatoGuy** [comment]
I narrowed down the hanging in #8111 to be from the updater thread not being a daemon so it prevented the master thread from closing. I experienced it consistently while I had a network issue that timed out all of my updater checks. After I fixed the network issue, I noticed it would only hang if I closed the game immediately after opening, before the new update notification appeared in the top right. I encountered this in both windows and debian (4.9.88). 

So if you're closing the game quickly, then this is probably what is happening. If its hanging even after an hour long session, then it *might* be something else but it could still be a network issue - my network issue was specifically timing out the updater and not affecting anything else on my computer.

*Note that if you are on the latest version, the updater check still runs even though no notif will end up showing, so it may not be obvious when the thread completes.
